tests/e2e/volumepropverification_test.go

```diff
@@ -33,14 +33,14 @@ import (
 )
 
 var (
-	esx              string = os.Getenv("ESX")
+	esx               = os.Getenv("ESX")
 	volSizes                = []string{"100MB"}
 	vms                     = []string{os.Getenv("VM1")}
```

Comment:
The idea was to run against multiple vms (having it said vm1 is photon and vm2 is ubuntu) ... go testframework is having timeout of 10 mins and there were occurrences this simple test was going over 10 mins causing failure for the PR. So we have turned off running multiple VMs.

Performance improvement is tasked out to leverage running this test against multiple inputs.

for `volSizes`: The test was running against 100MB, 1GB, 1TB due to above issue we trimmed down running against only one input. 

> This has caused 2 unnecessary nested for-loop in TestVolumeProperties().
Kept the for loop to turn on extra combination in the future.